woocommerce/google-listings-and-ads
# Fix validation for empty shipping rates and shipping times

### Changes proposed in this Pull Request:

Closes #1435.

This PR modifies some validation handling for shipping rates and shipping times in Setup MC and Edit Free Listings to fix the issue #1435.

### Screenshots:

📹  Demo video with my voice:

https://user-images.githubusercontent.com/417342/166510966-7ca9d85a-1c14-4c94-be9f-a616829eafe8.mp4

### Detailed test instructions:

1. Go to Setup MC Step 3.
2. Delete all the shipping rates and shipping times.
3. Click on the shipping rate "Edit" button to open the edit modal. The update button in the modal should be disabled.
4. Click on the shipping time "Edit" button to open the edit modal. The update button in the modal should be disabled.
5. The "Continue" button at the bottom of page should only be enabled when you have all the required valid values for shipping rates and shipping times.
6. Repeat the above steps for Edit Free Listings.


### Changelog entry

> Fix - Validation for shipping rates and shipping times in Setup MC and Edit Free Listings.
